cmd/ssh.go

```diff
@@ -243,7 +244,7 @@ func (cmd *SSHCmd) startTunnel(ctx context.Context, devPodConfig *config.Config,
 	defer writer.Close()
 
 	log.Debugf("Run outer container tunnel")
-	command := fmt.Sprintf("'%s' helper ssh-server --track-activity --stdio", agent.ContainerDevPodHelperLocation)
+	command := fmt.Sprintf("'%s' helper ssh-server --track-activity --stdio --workdir '%s'", agent.ContainerDevPodHelperLocation, filepath.Join("workspaces", workspaceName))
```

Comment:
On second thought `filepath.Join("workspaces", workspaceName)` is probably not right... What would be the right way to get the directory?

Reply:
`ContainerWorkspaceFolder` is the only thing I could find but it seems available at a very different point in the lifecycle.

https://github.com/loft-sh/devpod/blob/82e870c14b8834d3b3fedee3e1e94364b836e4a9/pkg/devcontainer/config/substitute.go#L26

Reply:
You will need to make workspace dir absolute `/workspaces`. 
I guess it's fine as a default, when users set `workspaceFolder` in the `devcontainer.json`, it won't be able to find this path and fall back to the default

Reply:
yeah but we don't have a parsed `devcontainer.json` if I'm not mistaken

Reply:
That seems correct to me there doesn't seem to be a parsed devcontainer.json, only a devpod config.